setup.sh

```diff
@@ -1,13 +1,26 @@
-#!/bin/bash
-read -n 1 -p "Specify python version(2/3)(Default-3)" answer
-echo ""
-case ${answer:0:1} in
-2 )
-    echo "Selected python version 2"
-;;
-* )
-    echo "Selected python version 3"
-;;
+#!/bin/bash -xe
+
+case ${1} in
```

Comment:
It may not be worth the effort, but I think that if you create a docker image for jarvis, it should not use the `setup.sh`  script since environment is well controlled compared to bare metal machine. 

Instead, iI suggest you install deps in Dockerfile, and provide several Dockerfiles if you wish to support several distributions (which may not be necessary). This way you will reduce drastically image size (no virtualenv and no cache when installing with apt or pip).